### PR TITLE
Add info for how to execute arc-script

### PR DESCRIPTION
**Pull request by segeljakt**
A small README update.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,6 +41,10 @@ cd arc-mlir/; ./arc-to-mlir-build; cd -
 
 # Run tests
 cd arc-mlir/build/llvm-build/; ninja check-arc-mlir; cd -
+
+# Compile and run arc-script
+cd arc-script/
+cargo run --release --features=repl -- --check repl
```
Comment (frej) [inline README.md:47]:
```
$ cargo run --release --features=repl -- --check repl
error: --features is not allowed in the root of a virtual workspace
note: while this was previously accepted, it didn't actually do anything
```

Removing `--features=repl` gives:
```
 $ cargo run --release -- --check repl
    Finished release [optimized] target(s) in 0.09s
     Running `target/release/arc-script --check repl`
error: Found argument 'repl' which wasn't expected, or isn't valid in this context

If you tried to supply `repl` as a PATTERN use `-- repl`

USAGE:
    arc-script [FLAGS] [OPTIONS] <SUBCOMMAND>

For more information try --help
frej@emu .../arc/arc-script $
```

This is with cargo and rustc 1.46.0-nightly.

Comment (segeljakt) [reply README.md:47]: Ah forgot about the workspace.

Comment (segeljakt) [reply README.md:47]: Or did you run it from the current commit? It is strange that you got the first message, there should not be a workspace on this branch.

Comment (frej) [reply README.md:47]: I merged arc-script into mlir, and ran it there.

Comment (segeljakt) [reply README.md:47]: Ok, now it should work. I can squash the commits if we merge.

Comment (frej) [reply README.md:47]: Those instructions work, please merge the two commits. If things are different between the mlir and the arc-script branches, move this change to the arc-script branch.

**Comment by segeljakt** [comment]
I will close this and address it in a new PR

**Pull request closed without merging**